examples/webpack/README.md

```diff
@@ -21,17 +21,14 @@ the worker code, and the `workerSrc` path shall be set to the latter file.
 ### Minification
 
 If you are configuring Webpack to output a minified build, please note that you
-*must* configure the minifier to keep original class/function names intact;
+_must_ configure the minifier to keep original class/function names intact;
```

Comment:
This looks like an unrelated change, since the word is already italicized when looking at https://github.com/mozilla/pdf.js/tree/master/examples/webpack#minification; please revert this change.

Reply:
This change was made automatically by my Prettier extension. If I revert the change, the next person who edits the file may encounter the same issue. Do you still want me to revert it?

Reply:
As a general rule, we *try* to avoid making unrelated changes in patches.

> This change was made automatically by my Prettier extension.

Sorry, but considering that we're not using such a thing in this project that's maybe not entirely relevant here.

If the suggestion is that we should start using such a plugin, then that's a discussion that ought to happen elsewhere.
(For that sort of changes to be considered though, such a task *should* run as part of `gulp lint` and we'd also want to change *all* README files at once rather than piecemeal like this.)

> Do you still want me to revert it?

Yes, otherwise I'd not have comment on this :-)